README.md

```diff
@@ -13,18 +13,13 @@
 ### Technologies used 
 - Git, linters and Github  
 
```

Comment:
I am requesting these changes here because I am unable to highlight the code.

- [ ] In your `modules/bookScript.js` file, kindly refactor the following functions to arrow functions:

  - `callRemoveBook`, `callAddBook`, `retrieveBookData`, `addBookOnPage`

- [ ] In your `modules/DateTimeScript.js` file, kindly refactor this function to an arrow function:

  - `getDateTime`